### DISCUSSION TITLE
alpha.74 says it is unreleased on website?

### DISCUSSION THREAD

**Discussion by cmdcolin** [facebook/docusaurus#4702] [Q&A] [answered]
Not sure if maybe I have a cached version but I think from the changelog it says alpha.74 is released but the website says alpha.74 is the next unreleased version https://docusaurus.io/docs/next/

**Comment by slorber** [comment, marked as answer]
Thanks, this is because sometimes our prod website deployment gets stuck due to Crowdin. Will look for a solution.

Manually triggered a redeploy, should be fixed soon.
